partiql-tests-data/eval/query/pivot.ion

```diff
@@ -53,8 +53,8 @@ pivot::[
     }
   },
   {
-    name:"pivotUnpivotWithWhereLimit",
-    statement:"PIVOT val AT 'new_' || name FROM UNPIVOT `{a:1, b:2, c:3, d:4, e:5, f: 6}` AS val AT name WHERE name <> 'b' AND val <> 3 LIMIT 3",
+    name:"pivotUnpivotWithWhereOrderByLimit",
+    statement:"PIVOT val AT 'new_' || name FROM UNPIVOT `{a:1, b:2, c:3, d:4, e:5, f: 6}` AS val AT name WHERE name <> 'b' AND val <> 3 ORDER BY val LIMIT 3",
```

Comment:
Similar to limit offset tests, should include an ORDER BY to deterministically get the specified elements back